tests/SideBySide/DataTypesFixture.cs

```diff
@@ -121,8 +121,8 @@ public DataTypesFixture()
   latin1 varchar(300) character set 'latin1' null,
   latin1bin varchar(300) character set latin1 collate latin1_bin null,
   cp1251 varchar(300) character set 'cp1251' null,
-  guid char(36) null,
-  guidbin char(36) binary null
+  guid char(36) character set 'utf8mb4' null,
+  guidbin char(36) binary character set 'utf8mb4' null
```

Comment:
Why was this change needed?

Reply:
OK, I see that the "baseline" code returns 108 for this column length (== 36 * 3, so probably defaulting to `utf8`?). IMO, this is a bug in Oracle's connector; the `ColumnSize` of a `char(36)` column should be reported as 36, no matter the encoding. 

MariaDB's failure is interesting... I wonder if you could work around them both for now by using `character set 'ascii'` (and an expected column size of 36)?

Reply:
I read the code and it is a special case for GUID. I decide to do something special for GUID length which is different with Orcale's connector, what's your idea?